code/game/machinery/doors/door.dm

```diff
@@ -149,26 +149,34 @@
 				var/mob/living/simple_animal/bot/B = user
 				B.door_opened(src)
 		else
+			if(pry_open_check(user))
```

Comment:
Placing it here before the animation will change functionality a bit. Before the door would scream in terror that you can't enter before you forced it open.

Reply:
It's there because even on master code, airlocks do not like being in two animation states at once. That's how you get the cursed airlock bug where it opens or closes instantly.